[2024-07-26T21:00:18.997+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: exemplo_email_operator.4-Bash scheduled__2024-07-26T20:00:00+00:00 [queued]>
[2024-07-26T21:00:19.004+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: exemplo_email_operator.4-Bash scheduled__2024-07-26T20:00:00+00:00 [queued]>
[2024-07-26T21:00:19.005+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-26T21:00:19.005+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2024-07-26T21:00:19.005+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-26T21:00:19.013+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): 4-Bash> on 2024-07-26 20:00:00+00:00
[2024-07-26T21:00:19.017+0000] {standard_task_runner.py:55} INFO - Started process 52412 to run task
[2024-07-26T21:00:19.019+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exemplo_email_operator', '4-Bash', 'scheduled__2024-07-26T20:00:00+00:00', '--job-id', '1680', '--raw', '--subdir', 'DAGS_FOLDER/exemplo_email_operator.py', '--cfg-path', '/tmp/tmpcygfqxt1']
[2024-07-26T21:00:19.020+0000] {standard_task_runner.py:83} INFO - Job 1680: Subtask 4-Bash
[2024-07-26T21:00:19.068+0000] {task_command.py:388} INFO - Running <TaskInstance: exemplo_email_operator.4-Bash scheduled__2024-07-26T20:00:00+00:00 [running]> on host 029011c5aea1
[2024-07-26T21:00:19.118+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caue.ausec@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exemplo_email_operator
AIRFLOW_CTX_TASK_ID=4-Bash
AIRFLOW_CTX_EXECUTION_DATE=2024-07-26T20:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-26T20:00:00+00:00
[2024-07-26T21:00:19.119+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-07-26T21:00:19.120+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'exit 1']
[2024-07-26T21:00:19.135+0000] {subprocess.py:86} INFO - Output:
[2024-07-26T21:00:19.149+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-07-26T21:00:19.161+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-07-26T21:00:19.178+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=exemplo_email_operator, task_id=4-Bash, execution_date=20240726T200000, start_date=20240726T210018, end_date=20240726T210019
[2024-07-26T21:00:19.196+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:152: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2024-07-26T21:00:19.197+0000] {email.py:268} INFO - Email alerting: attempt 1
[2024-07-26T21:00:21.518+0000] {email.py:280} INFO - Sent an alert email to ['caue.ausec@gmail.com']
[2024-07-26T21:00:23.566+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1680 for task 4-Bash (Bash command failed. The command returned a non-zero exit code 1.; 52412)
[2024-07-26T21:00:23.603+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-26T21:00:23.690+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
